.circleci/config.yml

```diff
@@ -463,17 +463,27 @@ jobs:
       - run: *install_deps
       - run:
           name: Build conda Linux packages
+          no_output_timeout: 240m
           command: |
               export PATH=$HOME/miniconda/bin:$PATH
               . activate habitat;
               cd habitat-sim
 
+              # install Docker
+              curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
```

Comment:
Any reason you aren't using the magic `- setup_remote_docker` per the [documentation](https://circleci.com/docs/2.0/building-docker-images/)?

Reply:
`setup_remote_docker: step is not compatible with machine executor. You can use Docker without a special step if the build image has it installed.` Looks, like using `setup_remote_docker ` you should use different executor other than we use now.

Reply:
@mathfac "It is possible to specify a different executor type for every job in your ‘.circleci/config.yml’ by specifying the executor type and an appropriate image. An image is a packaged system that has the instructions for creating a running environment. A container or virtual machine is the term used for a running instance of an image. For example, you could specify an executor type and an image for every job:" https://circleci.com/docs/2.0/executor-types/